lib/package-card.coffee

```diff
@@ -227,9 +236,12 @@ class PackageCard extends View
       @displayNotInstalledState()
 
   displayInstalledState: ->
-    if @newVersion
+    if @newVersion or @newSha
       @updateButtonGroup.show()
-      @updateButton.text("Update to #{@newVersion}")
+      if @newVersion
+        @updateButton.text("Update to #{@newVersion}")
+      else if @newSha
+        @updateButton.text("Update to ##{@newSha.substr(0, 8)}")
```

Comment:
Same nit as https://github.com/atom/apm/pull/518#discussion-diff-56237165 about the meaning of the hash prefix.